### PR TITLE
Fix Empty component icon proptype

### PR DESCRIPTION
**Pull request by cedricmessiant**
The icon prop type should match the type of Icon component's icon prop

### CODE REVIEW

**Comment by cedricmessiant** [comment]
@ptbrowne done!

**Comment by cedricmessiant** [comment]
Now it's much better, thanks :-)

**Comment by ptbrowne** [comment]
:shipit:

File: react/ContextHeader/index.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const ContextHeader = ({ icon, title, text, onClose, className }) => {
 }
 
 ContextHeader.propTypes = {
-  icon: PropTypes.node,
+  icon: iconPropType,
```
Comment (Crash--) [inline react/ContextHeader/index.jsx]: Hmm `icon: PropTypes.oneOfType([PropTypes.node, iconPropType]),` ?

Comment (Crash--) [reply react/ContextHeader/index.jsx]: Or we add node to the iconPropType ;)

Comment (cedricmessiant) [reply react/ContextHeader/index.jsx]: as `icon` prop is directly passed to `Icon` component, I think it shouldn't be a node.

**Pull request merged**